python/hail/representation/annotations.py

```diff
@@ -70,3 +71,29 @@ def get(self, item, default=None):
         :returns: Value of item if found, or default value if not.
         """
         return self._attrs.get(item, default)
+
+
+@typecheck(struct=Struct)
+def to_dict(struct):
```

Comment:
basically `recurse=False` would be the same as _attrs, I guess, though would do a copy.